### PR TITLE
Html package

### PR DESCRIPTION
**Pull request by zorkow**
Port of MJ2's HTML package.

### CODE REVIEW

File: mathjax3-ts/input/tex/html/HtmlConfiguration.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+/**
+ * @fileoverview Configuration file for the Html package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import {CommandMap} from '../SymbolMap.js';
+import ParseMethods from '../ParseMethods.js';
```
Comment (dpvc) [inline mathjax3-ts/input/tex/html/HtmlConfiguration.ts]: Is `ParseMethods` needed here?

File: mathjax3-ts/input/tex/html/HtmlConfiguration.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+new CommandMap('html_macros', {
+  href:    'Href',
+  'class': 'Class',
+  style:   'Style',
+  cssId:   'Id'
+}, HtmlMethods);
+
+export const HtmlConfiguration = Configuration.create(
+  'html',
+  {handler: { macro: ['html_macros']}});
```
Comment (dpvc) [inline mathjax3-ts/input/tex/html/HtmlConfiguration.ts]: Can the `);` be moved to the next line?  (In order to end the indentation.)

File: mathjax3-ts/input/tex/html/HtmlConfiguration.ts (hunk 1 above)
Comment (zorkow) [reply mathjax3-ts/input/tex/html/HtmlConfiguration.ts]: No, removed.

File: mathjax3-ts/input/tex/html/HtmlConfiguration.ts (hunk 2 above)
Comment (zorkow) [reply mathjax3-ts/input/tex/html/HtmlConfiguration.ts]: Done.

**Pull request merged**